configs/webpack.css-assets.dev.config.js

```diff
@@ -6,11 +6,14 @@ module.exports = {
 
     mode: 'production',
 
-    entry: join(cwd(), 'src', 'styles', 'index.js'),
+    entry: {
+        'ux-aspects': join(cwd(), 'docs', 'styles', 'ux-aspects.js'),
+        'hpe-icons': join(cwd(), 'docs', 'styles', 'hpe-icons.js')
+    },
```

Comment:
I think the approach you had before with the two requires in a single js file was correct, it was just in the wrong place. I don't think these webpack changes are needed.

Reply:
This script is only used in development mode for the plunkers. If there is only one file then it will only output one stylesheet - ux-aspects.css. However in a production mode build there are two stylesheets, ux-aspects.css and the new hpe-icons.css file. So this change was required to have the dev and production environments the same. 

In other words the plunker needs two stylesheet imports in production as they are now separate stylesheets eg:

```html
<link rel="stylesheet" href="/css/ux-aspects.css">
<link rel="stylesheet" href="/css/hpe-icons.css">
```

But without this change the in the dev environment there would be no `hpe-icons.css` file.

Perhaps you have a better suggestion? This was to just to workaround that issue.

Reply:
I'm not entirely sure if I understand the problem, but we currently use CopyWebpackPlugin to populate the plunker assets directory. The hpe-icons CSS should be present on disk already since it's not part of the webpack build, so we can just treat it as a static asset as far as I know.

Reply:
Yes, that valid solution, probably better as hpe-icons file is unlikely to change during active development. The issue was just that we need the hpe-icons file to be present in dev mode, so this should work too